pkg/distro/distro.go

```diff
@@ -130,6 +130,15 @@ type ImageOptions struct {
 	Subscription     *subscription.ImageOptions `json:"subscription,omitempty"`
 	Facts            *facts.ImageOptions        `json:"facts,omitempty"`
 	PartitioningMode disk.PartitioningMode      `json:"partitioning-mode,omitempty"`
+
+	OutputFilename string `json:"output_filename"`
+}
+
+func (i *ImageOptions) Filename(imgType ImageType) string {
+	if i.OutputFilename != "" {
```

Comment:
Hm, if we do this we will have to add validation here too, something like:
```go
        if filepath.Base(s) != s {
		return fmt.Errorf("cannot use %q: only relative filenames supported", s)
	}
	if filepath.Clean(s) != s {
		return fmt.Errorf("cannot use %q: only clean filenames supported", s)
	}
``` 
and of course error handling which will make the diff a bit more annoying.